jung-api/src/main/java/edu/uci/ics/jung/graph/DelegateCTree.java

```diff
@@ -206,13 +211,12 @@ public boolean removeNode(N node) {
     if (!nodes().contains(node)) {
       return false;
     }
+    // TODO: Replace `Graphs.reachableNodes` with `Traverser.forTree.breadthFirst` for lazy eval?
```

Comment:
Seems like a good idea.  Mind filing an issue for this so that we don't lose track of it?

Reply:
Done! See #203.